src/VersionInfo.php

```diff
@@ -31,7 +31,7 @@
 
 final class VersionInfo{
 	public const NAME = "PocketMine-MP";
-	public const BASE_VERSION = "5.16.1";
+	public const BASE_VERSION = "5.17.1";
```

Comment:
5.17.0 no?

Reply:
nope, the tag wil be made in commit daaf08c562389bb36724ee93691805d9326f06f1